elements/pfe-select/demo/pfe-select.story.js

```diff
@@ -0,0 +1,115 @@
+import { storiesOf } from "@storybook/polymer";
```

Comment:
@kylebuch8 hey I've added storybook as well. Please let me know if this is okay and covers everything.
Also, I'm facing an issue with the markup on the storybook HTML. The tags are not indented and I'm not sure why that might be happening. Can you take a look and suggest something?

Reply:
@kylebuch8 i've updated the code for storybook and have added read me files as we discussed. The markup issue is still not resolved for me.

Reply:
@diwanshi - I think the markup issue is fine for now.